config/config.go

```diff
@@ -295,8 +295,18 @@ type DeploymentConfig struct {
 // defaults to the namespace configured for IstioNamespace (which itself defaults to 'istio-system').
 type IstioComponentNamespaces map[string]string
 
+// Feature flags available from the CR
 type KialiFeatureFlags struct {
-	IstioInjectionAction bool `yaml:"istio_injection_action,omitempty" json:"istioInjectionAction"`
+	IstioInjectionAction bool       `yaml:"istio_injection_action,omitempty" json:"istioInjectionAction"`
+	UIDefaults           UIDefaults `yaml:"ui_defaults,omitempty" json:"uiDefaults,omitempty"`
+}
+
+// Describe default settings for the UI, like the default selected namespace or
+// auto refresh interval.
+type UIDefaults struct {
+	Namespaces          []string `yaml:"namespaces,omitempty" json:"namespaces,omitempty"`
+	MetricsPerRefresh   string   `yaml:"metrics_per_refresh,omitempty" json:"metricsPerRefresh,omitempty"`
+	AutoRefreshInterval string   `yaml:"auto_refresh_interval,omitempty" json:"autoRefreshInterval,omitempty"`
```

Comment:
Fine but even better if alphabetized.

Reply:
What are the "json:" fields for? I don't think we need those and this just makes things more verbose than necessary